Resources/Prototypes/Loadouts/neck.yml

```diff
@@ -102,3 +142,19 @@
   exclusive: true
   items:
     - ClothingNeckTransPin
+
+- type: loadout
+  id: LoadoutItemsAutismPin
+  category: Neck
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingNeckAutismPin
+
+- type: loadout
+  id: LoadoutItemsGoldAutismPin
```

Comment:
Lmao give this one an Atmos tech playtime requirement. I can think of no other department that better fits the bill.